features/dataset.yml

```diff
@@ -0,0 +1,13 @@
+name: Dataset
+description: "The `dataset` feature is for attaching and accessing custom data on elements, using either the `dataset` property or attributes prefixed with `data-`."
+spec: https://html.spec.whatwg.org/multipage/dom.html#embedding-custom-non-visible-data-with-the-data-*-attributes
```

Comment:
```suggestion
spec: https://html.spec.whatwg.org/multipage/dom.html#embedding-custom-non-visible-data-with-the-data-*-attributes
group: javascript
```

Reply:
So far the JS group has only been for JS language.